### PR TITLE
first version of hardware resources list

### PR DESCRIPTION
**Pull request by biphasic**
not complete yet, but a start

### CODE REVIEW

**Comment by neural-loop** [comment]
I am reviewing this, and will be looking into the UI around this content

**Comment by biphasic** [comment]
Thanks a lot @neural-loop . Any changes you need don't hesitate to ping me

File: content/english/hardware/SNP/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: SNP by Innatera
+description:  The spiking neural processor is Inatera's ultra-low-power neuromorphic chip for real-time edge intelligence. Using spiking neural networks, it achieves under 1mW operation for sensor applications like always-on audio scene detection with 85% accuracy in under 1ms.
```
Comment (fabrizio-ottati) [inline content/english/hardware/SNP/index.md:3]: Typo alert: 'Inatera'

**Comment by neural-loop** [comment]
I have merged the prior work for the hardware UX into this. I'll be doing some fine tuning on this to improve the display further. 
The data architecture will be 

- neuromorphic-computing/ landing page with information
  - hardware/ landing page with information
    - specific-hardware-item detail page
  - software/ landing page with information
    - specific-hardware-item detail page

**Comment by neural-loop** [comment]
It could be helpful if for one or two entries, t
* company / org info was fleshed out, with icon and social links, product page and such

if we want a carousel that would flip between multiple images too, I think we could do like the addition of a carousel/ folder that contains multiple images. It's not a hard requirement but I think it could give the pages a bit of life.

**Comment by neural-loop** [comment]
Having release date could be useful, I think probably it is a good sorting property, as more recently released hardware will probably be of most interest to most viewers. 

This will all be reworked (I put some AI generated content together to flesh out an initial page), but I'd like to put more detail into the list of hardware and sort them by date

I am splitting them into 2 lists "active efforts" and "legacy efforts". I'll probably make the hardware list use more width on the page and change the details that are shown for each. 

image

**Comment by biphasic** [comment]
amazing! Going to work on company info / socials for some entries. I also started work on the software

**Comment by biphasic** [comment]
I fleshed out Xylo, Speck and DynapCNN in particular, but also added more links to the other hardware

**Pull request closed without merging**